libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -499,6 +546,14 @@ bool AP_UAVCAN::try_init(void)
                         debug_uavcan(1, "UAVCAN BatteryInfo subscriber start problem\n\r");
                         return false;
                     }
+					
+					uavcan::Subscriber<uavcan::equipment::esc::Status> *esc_st;
+					esc_st = new uavcan::Subscriber<uavcan::equipment::esc::Status>(*node);
```

Comment:
no check for nullptr ?

Reply:
No, no check. If we can run out of mem, then such check should be added to all other lines here during init.